### ISSUE TITLE
not an issue / deleted

### ISSUE THREAD

**Issue by varoudis** [fchollet/keras#3743]
Is this really the appropriate loss function when reconstruction target is a scales float32?
That puzzles me a bit.
Am I missing something?
